tools/challenge-editor/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@freecodecamp/challenge-editor",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run start:api & npm run start:client",
+    "start:api": "cd api && npm start",
+    "start:client": "cd client && npm start"
+  },
+  "author": "",
+  "license": "BSD-3-Clause"
+}
```

Comment:
Since this just holds some scripts, I think it would be better to pull those scripts to the root and delete this.  Then contributors don't need to go hunting for this.